pkgs/development/python-modules/rising/default.nix

```diff
@@ -60,5 +73,7 @@ buildPythonPackage rec {
     homepage = "https://rising.rtfd.io";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ bcdarwin ];
+    # AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
+    # broken = pythonAtLeast "3.12";
```

Comment:
Does this still apply?

Reply:
Nope, I forgot to remove it.
Now it is gone.